dll/win32/setupapi/parser.c

```diff
@@ -1235,16 +1232,23 @@ HINF WINAPI SetupOpenInfFileW( PCWSTR name, PCWSTR class, DWORD style, UINT *err
 
     TRACE("%s %s %lx %p\n", debugstr_w(name), debugstr_w(class), style, error);
 
+#ifdef __REACTOS__
     if (style & ~(INF_STYLE_OLDNT | INF_STYLE_WIN4))
     {
         SetLastError(ERROR_INVALID_PARAMETER);
         return (HINF)INVALID_HANDLE_VALUE;
     }
+    if (!name)
+    {
+        SetLastError(ERROR_INVALID_PARAMETER);
+        return (HINF)INVALID_HANDLE_VALUE;
+    }
```

Comment:
```suggestion
    if (style & ~(INF_STYLE_OLDNT | INF_STYLE_WIN4))
    {
        SetLastError(ERROR_INVALID_PARAMETER);
        return (HINF)INVALID_HANDLE_VALUE;
    }

    if (!name)
    {
        SetLastError(ERROR_INVALID_PARAMETER);
        return (HINF)INVALID_HANDLE_VALUE;
    }
```
Or you could merge these two ifs.